Tests/AwesomeAssertions.Specs/Primitives/StringAssertionSpecs.EndWith.cs

```diff
@@ -112,6 +127,282 @@ public void When_string_ending_is_compared_and_actual_value_is_null_then_it_shou
             act.Should().Throw<XunitException>().WithMessage(
                 "Expected someString to end with \"ABC\", but found <null>.");
         }
+
+        [Fact]
+        public void When_short_strings_have_no_common_chars_arrows_are_aligned()
+        {
+            // Act
+            Action act = () => "A".Should().EndWith("B");
+
+            // Assert
+            act
+                .Should()
+                .Throw<XunitException>()
+                .WithMessage(
+                    """
+                    *index 0*
+                       ↓ (actual)
+                      "A"
+                      "B"
+                       ↑ (expected).
+                    """);
+        }
+
+        [Fact]
+        public void When_one_string_is_long_and_the_other_is_short_arrows_are_aligned()
+        {
+            // Arrange
+            const string subject = "this is a long text that has more than 60 characters so it requires ellipsis";
+            const string expected = "requires an ellipsis";
+
+            // Act
+            Action act = () => subject.Should().EndWith(expected);
+
+            // Assert
+            act
+                .Should()
+                .Throw<XunitException>()
+                .WithMessage(
+                    """
+                    *index 66*
+                                     ↓ (actual)
+                      "…so it requires ellipsis"
+                          "requires an ellipsis"
+                                     ↑ (expected).
+                    """);
+        }
+
+        [Fact]
+        public void When_expected_is_short_strings_are_right_aligned()
+        {
+            // Act
+            Action act = () => "ABCDEFGHI".Should().EndWith("H");
+
+            // Assert
+            act
+                .Should()
+                .Throw<XunitException>()
+                .WithMessage(
+                    """
+                    *before*index 8*
```

Comment:
```suggestion
                    *before index 8*
```
When checking `before index` we should not use the wildcard in-between.